templates/docker-compose/Makefile.dist

```diff
@@ -13,7 +14,7 @@ help:
 
 .PHONY: install
 install: configure create_secrets create_volumes ## Install the helpdesk.
-	$(DOCKER_COMPOSE_BIN) $(COMPOSE_FILES) up -d gateway supportpal db redis
+	$(DOCKER_COMPOSE_BIN) $(COMPOSE_FILES) up -d supportpal
```

Comment:
How does this work? Those services need to be up?

Reply:
At this point, we only want to kick off the command line installation, we only need the main web service, and docker-compose will also up the dependencies based on `depends_on`